src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,55 @@
 import React from 'react';
 import './TodoList.scss';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+import { Todo } from '../Todo/Todo';
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
+export const TodoList = ({ todos, selectUser, query, selection }) => {
+  let filteredTodos = todos
+    .filter(todo => todo.title.toLowerCase()
+      .includes(query.toLowerCase()));
 
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
+  if (selection === 'Select completed') {
+    filteredTodos = filteredTodos.filter(todo => todo?.completed);
+  } else if (selection === 'Select not completed') {
+    filteredTodos = filteredTodos.filter(todo => !todo?.completed);
+  }
```

Comment:
It's better to make all sortings in upper level and pass already preparedTodos to 'TodosList' component